### PR TITLE
Fixes #11972: pre-inst scripts should be part of the package

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/11972

Replacing previous PR: https://github.com/Normation/rudder-packages/pull/1447

### CODE REVIEW

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -338,7 +338,65 @@ then
...
+
+# Restart daemons if we stopped them, otherwise not
+if [ ${CFRUDDER_FIRST_INSTALL} -ne 1 ]
+then
+  # Part of the package now, but create it anyway in case
+  # of upgrade from a very old version
+  mkdir -p /var/rudder/tmp
+
+%if "${use_systemd}" == "true"
+  then
```
Comment (peckpeck) [inline rudder-agent/SPECS/rudder-agent.spec]: no then with %if

File: rudder-agent/SPECS/rudder-agent.spec [hunk 2]
```diff
@@ -338,7 +338,65 @@ then
   CFRUDDER_FIRST_INSTALL=1
```
Comment (peckpeck) [inline rudder-agent/SPECS/rudder-agent.spec]: CFRUDDER_FIRST_INSTALL=$1  ?

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
@peckpeck updated

**Pull request merged**